crates/biome_js_analyze/src/lint/complexity/no_useless_fragments.rs

```diff
@@ -158,14 +160,20 @@ impl Rule for NoUselessFragments {
                                     parent.kind(),
                                     JsSyntaxKind::JS_RETURN_STATEMENT
                                         | JsSyntaxKind::JS_INITIALIZER_CLAUSE
-                                        | JsSyntaxKind::JS_CONDITIONAL_EXPRESSION
                                         | JsSyntaxKind::JS_ARROW_FUNCTION_EXPRESSION
                                         | JsSyntaxKind::JS_FUNCTION_EXPRESSION
                                         | JsSyntaxKind::JS_FUNCTION_DECLARATION
                                         | JsSyntaxKind::JS_PROPERTY_OBJECT_MEMBER
                                 )
                             }),
-                        Err(_) => JsxAttributeInitializerClause::try_cast(parent.clone()).is_ok(),
+                        Err(_) => {
+                            if let Ok(_) = JsxChildList::try_cast(parent.clone()) {
```

Comment:
It looks like Clippy is still complaining about this line.

Reply:
yeah, i just update here~